ts/store/reducers/navigation.ts

```diff
@@ -32,7 +32,8 @@ export function navigationReducer(
 
 /**
  * For debug / backwards compatibility purpose only
- * @deprecated Don't use this selector to create new application logic, will be removed after the upgrade to react-navigation v6
+ * The use of this selector is highly discouraged, it should be better to handle the current route through react-navigation proper hooks.
```

Comment:
Shall we remove the `Don't use this selector ...` since we have already upgraded to react-navigation v6 and made more usages of the selector?

Reply:
Actually agree, we should anyway find a proper solution to be able to rerender some components based on the navigation status change and avoid storing the currentRoute on redux, I didn't found any better solution by far, so this selector actually maybe the only available solution we have. Maybe we can change the comment to enphasize the need to discover better solutions avoiding the redux one.. let's remove the comment, at the moment.